static/xref/filter/worker.js

```diff
@@ -0,0 +1,68 @@
+let data = [];
```

Comment:
why "global" data?

Reply:
We fetch once on start, then reuse that data on all further queries